views/spots/view.njk

```diff
@@ -19,6 +19,9 @@
     {% if spot.description %}
     <p class="description">{{ spot.description }}</p>
     {% endif %}
+    {% if spot.payload %}
+    <pre>{{ spot.payload }}</pre>
+    {% endif %}
```

Comment:
Same comment as before, I would move this to a new drop down menu "Display advanced fields (mostly for developers)" to avoid confusion.

For the time being, I would at least add a title "JSON Payload" and move it under the map as the current version is a bit confusing for the end user so they can focus on the content that is important for them.